benchopt/base.py

```diff
@@ -132,6 +132,10 @@ def install(cls, env_name=None, force=False):
                     pip_install_in_env(*cls.requirements_install,
                                        env_name=env_name)
                 elif cls.install_cmd == 'bash':
+                    # eventually install other packages needed
+                    if hasattr(cls, "requirements_import"):
```

Comment:
This should be `requirements`, `requirements_import` is a hook for when the name of the import is different from the name of the `requirement`.